File: Begin/src/Begin19.java
/*
Begin19. To'g'ri to'rtburchakning qarama-qarshi uchlari koordinatlari berilgan.
Uning tomonlari koordinata o'qiga parallel. To'g'ri to'rtburchakning perimetri va yuzasi aniqiansin.
* */
import java.util.Scanner;
public class Begin19 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("A(x1,y1) = ");
        int x1 = in.nextInt();
        int y1 = in.nextInt();
        System.out.print("B(x2,y2) = ");
        int x2 = in.nextInt();
        int y2 = in.nextInt();
        int A = (x2-x1);
        int B = (y2-y1);
        System.out.println("P = "+(2*(A+B))+" S = "+A*B);

    }
}

File: Begin/Begin_19.py
print("Togri tortburchakning qarama-qarshi uchlari koordinatlarini kiriting:")
#uning tomonlari koordinata o'qiga parallel, To'g'ri to'rtburchakning perimetri va yuzasini aniqlang
#A(x1,y1) D(x2,y2) AB==CD==x2-x1 va AC==BD==y2-y1
# A________________B
# |               |
# |               |
# |               |
# |_______________|
# C               D
x1=int(input("x1="))
y1=int(input("y1="))
x2=int(input("x2="))
y2=int(input("y2="))
AB=abs(x2-x1)
BD=abs(y2-y1)
print("To'g'ri to'rtburchak  \n perimetri: P=",2*(AB+BD),"\n Yuzasi: S=",AB*BD)